assignments/function-conversion.js

```diff
@@ -1,30 +1,29 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction = () => console.log("Function was invoked!");
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+myFunction();
 
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
+let anotherFunction = (param) => {return param;
+}
```

Comment:
Recommendation: To format your code and keep track of your curly braces, these extensions are recommended:

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer